input/HSCDR1wl__0.yml

```diff
@@ -18,5 +18,7 @@ zlim_nz: [0.0, 4.0]
 mask_name: 'mask_HSC_wl0'
 mapper_class: 'MapperHSCDR1wl'
 path_rerun: '/mnt/extraspace/damonge/Datasets/HSC_DR1/lite/'
+beam_info: 
+  - type: 'PixWin'
```

Comment:
you don't have to change anything, but just to remind ourselves that we actually need to remove this if the pixels are very small.

Reply:
nope